content/en-us/reference/engine/classes/EditableMesh.yaml

```diff
@@ -1065,8 +1231,10 @@ methods:
     security: None
     thread_safety: Unsafe
   - name: EditableMesh:Triangulate
-    summary: ''
-    description: ''
+    summary: |
+      Splits all faces on the mesh to be triangles.
+    description: |
+      Splits all faces on the mesh to be triangles. Currently this does nothing since only triangles can be created, but if your code relies on triangles it is good to call this method after calling `AssetService:CreateEditableMeshAsync`.
```

Comment:
```suggestion
      Splits all faces on the mesh to be triangles. Currently this does nothing since only triangles can be
      created, but if your code relies on triangles, it's recommended that you call this method after calling
      `Class.AssetService:CreateEditableMeshAsync()`.
```